src/table/settings/difficulty_settings.ini

```diff
@@ -98,9 +98,10 @@ type     = SLE_UINT32
 from     = SLV_97
 flags    = SF_NEWGAME_ONLY | SF_SCENEDIT_TOO | SF_GUI_CURRENCY
 def      = 300000
-min      = 0
+min      = LOAN_INTERVAL
 max      = 2000000000
-interval = 50000
+pre_cb   = [](auto new_value) { new_value = (new_value + LOAN_INTERVAL / 2) / LOAN_INTERVAL * LOAN_INTERVAL; return true; }
```

Comment:
`auto &` to make it more explicit? Kinda looks like it's modifying a local variable at the moment otherwise

Reply:
Sure, but then you'd get a few other instances as bonus ;)